assets/css/fonts.css

```diff
@@ -179,3 +179,23 @@
     url(../../node_modules/bootstrap-italia/dist/fonts/Roboto_Mono/RobotoMono-Bold.ttf) format('truetype');
     unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
 }
+.megamenu{
```

Comment:
Remove all this CSS. The only CSS allowed is the BI package.

Reply:
So what should I use instead of CSS? If I have to set the width of container?